packer/windows.pkr.hcl

```diff
@@ -0,0 +1,154 @@
+variable "image_date" { type = string }
+variable "gcp_project" { type = string }
+variable "task_name" { type = string }
+
+variable "prefix" {
+  type = string
+  default = ""
+}
+
+locals {
+  name = "${var.prefix}pg-ci"
+
+  windows_gcp_images = [
+    {
+      name = "${var.task_name}"
+    },
+  ]
+}
+
+source "googlecompute" "windows-vm" {
```

Comment:
Have you tried to generate docker containers using packer as well? I'd like to have a somewhat orderly transition for CI, and that'll be harder if we just stop producing the containers on a certain date.

Reply:
I didn't try it yet, will work on that.